azurerm_virtual_hub_connection - support the `routing` block (#8950)
terraform-providers/terraform-provider-azurerm: website/docs/r/virtual_hub_connection.html.markdown

* `internet_security_enabled` - (Optional) Should Internet Security be enabled to secure internet traffic? Changing this forces a new resource to be created. Defaults to `false`.

* `routing_configuration` - (Optional)  A `routing_configuration` block as defined below.

---

> An `routing_configuration` block exports the following:

Author: updated

> * `associated_route_table_id` - (Optional) The ID of route table associated with Virtual Hub connection.

Reviewer: grammer
```suggestion
* `associated_route_table_id` - (Optional) The ID of the route table associated with this Virtual Hub connection.
```

Author: updated

> * `associated_route_table_id` - (Optional) The ID of route table associated with this routing configuration.

Author: updated

* `propagated_route_table` - (Optional)  A `propagated_route_table` block as defined below.

* `vnet_static_route` - (Optional)  A `vnet_static_route` block as defined below.

---

> An `propagated_route_table` block exports the following:

Author: updated

> * `labels` - (Optional) The list of labels.

Author: updated

> * `labels` - (Optional) The list of labels to provide a mechanism to logically group route tables.

Reviewer: ```suggestion
* `labels` - (Optional) The list of labels to assign to this route table.
```

Author: updated

> * `route_table_ids` - (Optional) The list of resource ids of all the route tables.

Author: updated

---

> An `vnet_static_route` block exports the following:

Author: updated

> * `route_table_ids` - (Optional) A list of Route Table ID's which is used for Virtual Hub Connection.

Author: updated

> * `name` - (Optional) The name which should be used for this static_route.

Author: updated

> * `address_prefixes` - (Optional) The list of all address prefixes.

Author: updated

> * `next_hop_ip_address` - (Optional) The ip address of the next hop.

Author: updated

> * `address_prefixes` - (Optional) A list of CIDR Blocks which should be used for Virtual Hub Connection.

Author: updated
